### PR TITLE
CSI-3491 update svc expand command

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: controller/array_action/array_mediator_svc.py [hunk 1]
```diff
@@ -317,10 +317,13 @@ def _get_object_fcmaps(self, object_name):
         all_fcmaps.extend(self._get_fcmaps_as_source_if_exist(object_name))
         return all_fcmaps
 
-    def _expand_cli_volume(self, cli_volume, increase_in_bytes):
+    def _expand_cli_volume(self, cli_volume, increase_in_bytes, is_in_remote_copy_relationship):
         volume_name = cli_volume.name
         try:
-            self.client.svctask.expandvdisksize(vdisk_id=volume_name, unit='b', size=increase_in_bytes)
+            if is_in_remote_copy_relationship:
+                self.client.svctask.expandvolume(object_id=volume_name, unit='b', size=increase_in_bytes)
```
Comment (ArbelNathan) [inline controller/array_action/array_mediator_svc.py:324]: as I understand, `expandvolume` is only for HyperSwap but `is_in_remote_copy_relationship` implies that its also for replication..

Comment (oriyarde) [reply controller/array_action/array_mediator_svc.py:324]: this is the existing check we use to delete a hyperswap volume.
I agree this existing function name might be misleading.
unlike hyperswap, replication (mirroring) does not use fcmaps, so no functional gap here

Comment (ArbelNathan) [reply controller/array_action/array_mediator_svc.py:324]: so PVC that is a part of a Replication object is not in a remote copy relationship?

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**